pornhub/videos.py

```diff
@@ -56,6 +56,37 @@ def _craftVideosURL(self, page_num, sort_by):
 
         return payload
 
+    def _loadPage(self, page_num=None, sort_by=None, url=None, viewkey=None):
+        
+        # load search page
+        if page_num:
+            if self.keywords:
+                r = requests.get(BASE_URL + VIDEOS_URL + SEARCH_URL, params=self._craftVideosURL(page_num, sort_by), headers=HEADERS, proxies=self.ProxyDictionary)
+            else:
+                r = requests.get(BASE_URL + VIDEOS_URL, params=self._craftVideosURL(page_num, sort_by), headers=HEADERS, proxies=self.ProxyDictionary)
+        # load video page
+        else:
+            if url and isVideo(url):
+                r = requests.get(url, headers=HEADERS, proxies=self.ProxyDictionary)
+            else:
+                r = requests.get(BASE_URL + VIDEO_URL + viewkey, headers=HEADERS, proxies=self.ProxyDictionary)
+        
+        html = r.text
+        
+        return BeautifulSoup(html, "lxml")
+
+    """
+    def _loadVideoPage(self, url, viewkey):
```

Comment:
So `_loadVideoPage` is replaced with `_loadPage`?

Reply:
I combined 2 page loading methods into 1 _loadPage

Reply:
Because `_loadVideoPage` and `loadVideosPage` do essentially the same thing, load the page.
I think this combination makes sense

Reply:
Feel free to delete that chunk of code completely.